com.openmusicplayer.OMP.json

```diff
@@ -0,0 +1,257 @@
+{
+
+  "app-id": "com.openmusicplayer.OMP",
+
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.30",
+
+  "sdk": "org.gnome.Sdk",
+
+
+  "finish-args":
+  [
+
+    "--socket=x11",
+    "--share=network",
+    "--filesystem=host",
```

Comment:
Why do you need access to entire host filesystem?

Reply:
That was the only way I could figure out how to get access to all of the music files I use to test. Is there a better --filesystem option I should use than that?

Reply:
> That was the only way I could figure out how to get access to all of the music files I use to test. Is there a better --filesystem option I should use other than that?

Gtk+3 (and thus also gtkmm 3.x) and Qt5 apps can use portals, but I'm afraid this is not suitable for a music player with playlist support.
Anyway, @barthalion is known for limiting  filesystem access as much as possible, so he would probably suggest the `--filesystem=xdg-music`. In my opinion, it's just unnecessary inconvenience to users, and this app should at least have `--filesystem=home` permissions.

Reply:
What my dear friend @scx seems to be known for is notoriously ignoring the fact that permissions aren't set in stone and users that actually store files outside `xdg-music` can easily extend it with `flatpak override`. While it requires CLI now, it will be easily available from GNOME settings in next stable release and I'd surprised if KDE doesn't do it or plan so already.

Reply:
I don't even store my music in the XDG music directory. I would definitely prefer OMP have access to the whole system rather than a single folder by default, but I'll do whatever is required to get this published. I'll be working on all the changes suggested and required tomorrow and will make a commit then. Let me know if I should definitely make it xdg-music and I will.

Thanks for keeping such high standards. It makes me confident in my choice to use Flatpak.